storage/database/dynamodb.go

```diff
@@ -177,13 +187,18 @@ func (dynamo *dynamoDB) createTable() error {
 				KeyType:       aws.String("HASH"), // HASH - partition key, RANGE - sort key
 			},
 		},
-		// ProvisionedThroughput: &dynamodb.ProvisionedThroughput{
-		// 	dynamoReadCapacityUnits:  aws.Int64(dynamo.config.dynamoReadCapacityUnits),
-		// 	dynamoWriteCapacityUnits: aws.Int64(dynamo.config.dynamoWriteCapacityUnits),
-		// },
+
 		TableName: aws.String(dynamo.config.TableName),
 	}
 
+	if dynamo.config.IsProvisioned {
+		input.BillingMode = aws.String("PROVISIONED")
+		input.ProvisionedThroughput = &dynamodb.ProvisionedThroughput{
+			ReadCapacityUnits:  aws.Int64(dynamo.config.ReadCapacityUnits),
+			WriteCapacityUnits: aws.Int64(dynamo.config.WriteCapacityUnits),
+		}
+	}
```

Comment:
How about leaving a log about provisioned size as it is directly related to the money we need to pay.

Reply:
review reflected :)